packages/identitykit/src/lib/types.ts

```diff
@@ -8,7 +8,6 @@ export type SignerConfig = {
 }
 
 export enum TransportType {
-  POPUP,
```

Comment:
don't we need POPUP? seems heard that on call

Reply:
Dan asked me to rename POPUP into NEW_TAB. Frederik asked for POPUP, but the implementation is removed currently. If we decide to roll it back we probably do this under other ticket.